src/main/java/data/SheetsAuthUtil.java

```diff
@@ -21,25 +21,29 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.security.GeneralSecurityException;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.auth.oauth2.Credential;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.extensions.java6.auth.oauth2.AuthorizationCodeInstalledApp;
+import com.google.api.client.extensions.jetty.auth.oauth2.LocalServerReceiver;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.store.MemoryDataStoreFactory;
+import com.google.api.services.sheets.v4.SheetsScopes;
+
 
 /** 
- * A class to create and send an AppIdentityCredential object that allows for Sheets API interaction 
+ * A class to create and send an GoogleCredential object that allows for Sheets API interaction 
  */
 public class SheetsAuthUtil {
-  public static AppIdentityCredential authorize() throws IOException, GeneralSecurityException {
-         
-    // Build GoogleClientSecrets from JSON file.
-    InputStream in = SheetsAuthUtil.class.getResourceAsStream("/google-sheets-api.json");
-    GoogleClientSecrets clientSecrets = GoogleClientSecrets.load(
-      JacksonFactory.getDefaultInstance(), new InputStreamReader(in));
-
-    List<String> scopes = Arrays.asList(SheetsScopes.SPREADSHEETS);
- 
-    AppIdentityCredential credential = new AppIdentityCredential(scopes);
-
-    return credential;
+  public static GoogleCredential authorize() throws IOException, GeneralSecurityException {
+    return GoogleCredential.getApplicationDefault().createScoped(Collections.singleton(
```

Comment:
I'm a little confused, could you explain this?

Reply:
I'm going to have a PR where I delete this file. It is unnecessary at this point.